lib/tasks/remove_invalid_worldorg_pages.rake

```diff
@@ -0,0 +1,18 @@
+desc "Removes invalid about page drafts from world organisations that can prevent editing"
+task remove_invalid_worldorg_drafts: :environment do
```

Comment:
I think it would be good for this task to live in [the `data_hygiene` folder/namespace](https://github.com/alphagov/publishing-api/tree/master/lib/data_hygiene).

Reply:
I've moved it to the `data_hygiene` rake task file and currently testing the move was successful before updating this PR.